src/main/java/marquez/common/models/UrnType.java

```diff
@@ -14,9 +14,21 @@
 
 package marquez.common.models;
 
+import lombok.NonNull;
+
 public enum UrnType {
-  DATASET,
-  DATASOURCE;
+  DATASET(2),
+  DATASOURCE(2);
+
+  private final Integer value;
```

Comment:
nit: Value is too overloaded a term when it comes to enums - maybe we can call this `numParts` or something more descriptive?

Reply:
I'll be removing the enum (see above comment) and we can just use `numberOfParts` to communicate the number of parts in the URN.